<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationWin</class>
 <widget class="QWidget" name="RegistrationWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="Namelabel">
     <property name="text">
      <string>Name:</string>
     </property>
     <property name="buddy">
      <cstring>NameLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="NameLineEdit">
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="Namelabel_2">
     <property name="text">
      <string>Second Name: </string>
     </property>
     <property name="buddy">
      <cstring>SecondNameLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="SecondNameLineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="AccPassLabel">
     <property name="text">
      <string>Password: </string>
     </property>
     <property name="buddy">
      <cstring>PasswordEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="PasswordEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="Emaillabel">
     <property name="text">
      <string>Email: </string>
     </property>
     <property name="buddy">
      <cstring>EmailLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="EmailLineEdit"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="GenderLabel">
     <property name="text">
      <string>Gender</string>
     </property>
     <property name="buddy">
      <cstring>MradioButton</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QRadioButton" name="MradioButton">
     <property name="text">
      <string>Male</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QRadioButton" name="FradioButton">
     <property name="text">
      <string>Female</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QPushButton" name="RegisterButton">
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="IDdoc">
     <property name="text">
      <string>ID doctor:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="IDDocEdit"/>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="DepartmentEdit"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="Departmentlabel">
     <property name="text">
      <string>Department: </string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="PswdStrength">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QDateEdit" name="dateEdit">
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2020</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="displayFormat">
      <string>yyyy.M.d</string>
     </property>
     <property name="date">
      <date>
       <year>2020</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="RegistrationDateLabel">
     <property name="text">
      <string>Registration Date: </string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Registration Time</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QTimeEdit" name="timeEdit">
     <property name="displayFormat">
      <string>HH:mm:ss</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>NameLineEdit</tabstop>
  <tabstop>SecondNameLineEdit</tabstop>
  <tabstop>PasswordEdit</tabstop>
  <tabstop>IDDocEdit</tabstop>
  <tabstop>EmailLineEdit</tabstop>
  <tabstop>DepartmentEdit</tabstop>
  <tabstop>MradioButton</tabstop>
  <tabstop>FradioButton</tabstop>
  <tabstop>RegisterButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
